Library/Formula/wellington.rb

```diff
@@ -17,35 +17,58 @@ class Wellington < Formula
 
   depends_on "go" => :build
   depends_on "pkg-config" => :build
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
-  depends_on "libtool" => :build
 
-  go_resource "github.com/wellington/spritewell" do
-    url "https://github.com/wellington/spritewell.git",
-        :revision => "748bfe956f31c257605c304b41a0525a4487d17d"
+  stable do
+    depends_on "libsass"
   end
 
-  go_resource "github.com/go-fsnotify/fsnotify" do
-    url "https://github.com/go-fsnotify/fsnotify.git",
-        :revision => "f582d920d11386e8ae15227bb5933a8f9b4c3dec"
+  devel do
+    url "https://github.com/wellington/wellington/archive/c574754c39806e8c52682f15f49c8ff819d0f962.tar.gz"
```

Comment:
Libsass has been unusually quick to do releases lately, so wellington alpha is pointing to a stable release of libsass. In the future, it will need newer versions of libsass and it took a lot of trial and error to get these build steps in there right :). I wanted to save myself some effort in the future by keeping this in there.